meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/stm32mp15-eval/lmp.cfg

```diff
@@ -0,0 +1,42 @@
+CONFIG_BOOTCOMMAND="setenv bootargs android.serialno=${serial#} console=ttySTM0,115200 debug; stm32prog ${boot_device} ${boot_instance}; fastboot ${boot_device} ${boot_instance}"
```

Comment:
We may want to add a comment here to explain:
- We set boot args so that ADB has a serial # to refer to when using `adb devices` (and I think this is required)
- We start in `stm32prog` mode where the `STM32_Programmer_CLI` tool can be used
- After `STM32_Programmer_CLI -c port=USB1 -detach` is run we enter `fastboot` mode where further commands can be used.

Reply:
Added explanation to the commit message